Sources/InstanceStorage.swift

```diff
@@ -35,7 +35,7 @@ public final class GraphStorage: InstanceStorage {
         self.instance = instance
 
         if instances[graph] == nil { instances[graph] = Weak() }
-        instances[graph]?.value = instance
+        instances[graph]?.value = instance as AnyObject?
```

Comment:
Won't this result in nil when using the struct?

Reply:
Replicates the old behavior—references to structs were discarded.

Reply:
reverted to avoid any issues since this PR should still work without these changes